### PR TITLE
add cloudflare unique visitors for deno.land zone

### PR DESCRIPTION
**Pull request by wperron**

### CODE REVIEW

File: main.ts [hunk 1]
```diff
@@ -53,6 +60,54 @@ async function getThirdPartyModules(): Promise<number> {
...
+type CloudFlareAuth = {
+  email: string;
+  token: string;
+  zone: string;
+};
+
+async function getUniqueVisitors(cf: CloudFlareAuth): Promise<number> {
+  const now = new Date();
+  const yesterday = new Date(now);
+  yesterday.setDate(yesterday.getDate() - 1);
```
Comment (ry) [inline main.ts:72]: Will this work at the first of the month?

File: main.ts [hunk 2]
```diff
@@ -53,6 +60,54 @@ async function getThirdPartyModules(): Promise<number> {
...
+            uniq {
+              uniques
+              __typename
+            }
+            __typename
+          }
+        }
+        __typename
+      }
+    }`,
```
Comment (ry) [inline main.ts:94]: I don't know much about graphql ... but is this __typename necessary?

File: main.ts (hunk 1 above)
Comment (wperron) [reply main.ts:72]: according to the docs yes https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate

File: main.ts (hunk 2 above)
Comment (wperron) [reply main.ts:94]: No idea, I copied the query sent by the browsers and removed some stuff :shrug: I'll try without and if it works I'll remove it from the PR

**Comment by wperron** [comment]
Looking at the data, it is a bit weird because it always gets the number of unique visitors between now and 24h ago, so the number seems to be going up and down -- I'll put this one back in draft until I figure out a better way to insert the data

File: main.ts [hunk 3]
```diff
@@ -53,6 +62,80 @@ async function getThirdPartyModules(): Promise<number> {
...
+  return {
+    daily: body?.data?.viewer?.zones[0].daily[0].uniq?.uniques ?? 0,
+    monthly: body?.data?.viewer?.zones[0].monthly[0].uniq?.uniques ?? 0,
+  };
+}
+
+function toGQLDate(date: Date): string {
+  return `${date.getUTCFullYear()}-${
+    (date.getUTCMonth() + 1).toString().padStart(2, "0")
+  }-${date.getUTCDate().toString().padStart(2, "0")}`;
```
Comment (ry) [inline main.ts:136]: Maybe just 
```js
return date.toISOString().split('T')[0]
```

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**